src/App.tsx

```diff
@@ -20,12 +26,15 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={(event) => {
```

Comment:
It's better to move it to a separate function
Just FYI (take it into account in the future)